<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>150</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:#292C34;
color:white;
}</string>
  </property>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter the port number</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="portSelectionInput">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>256</number>
   </property>
   <property name="value">
    <number>3</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="windowsRadio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>121</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Windows (COM)</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="linuxRadio">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Linux /dev/ttyUSB</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testPortButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>89</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Test Port</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
